contracts/Conference.sol

```diff
@@ -18,15 +19,14 @@ contract Conference is GroupAdmin {
     mapping (uint => address) public participantsIndex;
 
     struct Participant {
-        uint participantIndex;
-        string participantName;
+        uint index;
         address addr;
         bool paid;
     }
 
-    event RegisterEvent(address addr, string participantName, uint participantIndex);
-    event FinalizeEvent(uint[] maps, uint256 _payout);
-    event WithdrawEvent(address addr, uint256 _payout);
+    event RegisterEvent(address addr, uint index);
```

Comment:
Any reason we use `uint` and `uint256`? Should be standardised to use `uint256` for consistency https://solidity.readthedocs.io/en/v0.4.21/types.html#integers

Reply:
Done